docs/preprocess.md

```diff
@@ -0,0 +1,30 @@
+# Preprocess
```

Comment:
It would be good to document what the advantages / disadvantages are of using this feature vs `svelte-preprocess`

Reply:
2 largest selling points are ts transform with esbuild and style url resolve. followed by general niceties of using the same css pipeline as vite, so less configuration.

Reply:
Good idea. `url()` isn't actually resolved in preprocessing as Vite will do another pass when it becomes a virtual module and resolve from there. This does make less configuration though so would be good to document.